components/script/dom/bindings/interface.rs

```diff
@@ -260,6 +261,12 @@ pub fn create_noncallback_interface_object(
     define_name(cx, rval.handle(), name);
     define_length(cx, rval.handle(), i32::try_from(length).expect("overflow"));
     define_on_global_object(cx, global, name, rval.handle());
+
+    if is_exposed_in(global, Globals::WINDOW) {
```

Comment:
This attribute is forbidden in non-Window globals so I'm not sure what this test is for.

Reply:
Interfaces with `[Exposed=(Window, Worker)]` can also have legacy window aliases, while they only actually get aliases on Window. This is to match the specified behavior.